Project-3/README.md

```diff
@@ -11,67 +11,238 @@ For any question involving the use of Protege, please be sure to import:
+
   (d) Goofus and Gallant have been married for three years; for each day of that span, it is true to assert they are married. 
+   Again, notice we are not modeling that Goofus and Gallant are married to each other.
+   M(x, t) = being married at t
+   Y(t) = belongs to 3 year span 1
+   g1 = Goofus
+   g2 = Gallant
+   D(t) = t is a day
+   ∀t(D(t) ∧ Y(t)→(M(g1,t) ∧ M(g2,t)))
+
 ```
 
 5. Using BFO and RO, model the following scenario: the content of an rdf file is represented in two serializations - one in Turtle, one in XML - which are sent from one computer to two distinct computers on the same network.   
 
+![Project3Q5](https://user-images.githubusercontent.com/106703400/223296934-5b8fa0bc-2f10-4b9a-a680-49a1164ce40a.png)
+
+(Comment: This is a graph based on Mermaid and the above blackboard work. Simply speaking, This graph represents the following key relations:
+
+– RDF File Content, Turtle and XML are all generically dependent continuants. RDF File Content has_model in Turtle and XML. And they are all carried by Computer.
+– Computer 1 enables the process of Data Transmission, which ends_with another process, Data Reception, which has computers 2 and 3 as participants. So this part is intended to show that there is some data (Turtle and XML) which are sent from Computers 1 to Computer 2 and 3.
+– Finally, computers 1, 2, and 3 are all parts_of network I, which is instance_of Object Aggregate.)
+
 
 6. Using Protege, place these in the BFO hierarchy where you think they fit best:
 ```
   (a) Bach's Well-Tempered Clavier
+    depending on whether we mean the music sheet/recording or the song itself, this could be classified as either a Generally Dependent Continuent or Process.
   (b) Chair of the UB Philosophy Department
+    Role
   (c) SARS-CoV-2
+    Object
   (d) Mexico City
+    I prefer to choose Site over Object Aggrogate. But 2D Spatial Region is the best choice.
   (e) The trunk of a minivan
+    Fiat Object Part
   (f) Occupation
+    Role
   (g) Ocean
+    Site is my top choice since the water level of the ocean is not of our concern, but to be safe, a 3D Spatial Region may be a good option as well. I am resisting to pick Object Aggrogate as an option
   (h) Lake
+    For the same reasons above, I will prefer to choose Site or 3D Spatial Region. A lake can go dry and will still be considered as a lake (Salt lake and Urumieh lakes in Iran)
+
 ```
 
 7. True or False; explain your answers:
 ```
   (a) An instance of Material Entity can have an instance of Immaterial Entity as part.
+    False. By definition, according to the axioms in BFO, Material Entity and Immaterial Entity are disjoint. Thus, there cannot be a single part of a material entity which an immaterial entity.
   (b) An instance of Immaterial Entity can have an instance of Material Entity as part.
+    False. According to the same axioms as mentions above, a Material Entity is disjoint with "part of some Material Entity" according to BFO.
```

Comment:
a **Material Entity** is disjoint with "part of some Material Entity" -- I guess that the first occurrence of "Material Entity"s should be "Immaterial Entity"? This should be a typo.